### ISSUE TITLE
Labelling of downloaded files - improvement?

### ISSUE THREAD

**Issue by nono-london** [jadchaar/sec-edgar-downloader#56]
Hello,
Thanks for this code.
I was wondering, if it was possible to have a more meaningful file labelling: "yyyy_mm_dd" +"filetype" for instance?
Best and thank you

**Comment by jadchaar** [comment]
Thanks for the feature request @nono-london! Since v4 is now using the Search API, obtaining such information is very easy and I should be able to add this naming scheme change in the future!

**Comment by Jennyyanhuadong** [comment]
I agree with 'nono-london'. An R package at https://cran.r-project.org/web/packages/edgar/edgar.pdf downloads files with labels that include cik, filing date, and accession number, such as '3116_8-K_2012-01-03_0001157523-11-007356'.

**Comment by ilhamait28** [comment]
Hello,
Thank you for the code!
I used the code to download the 10-k txt filings (without download_details) thanks to these lines:

from sec_edgar_downloader import Downloader
# Download filings to the current working directory
dl = Downloader()
# Get all 10-K filings for Microsoft without the filing details
dl.get("10-K", "MSFT", download_details=False)

I noticed that each txt file comes in a different folder, and all filings are labeled the same "full-submission.txt" which makes it hard to identify unless we check the folder's label itself.

Is there a possibility to have the txt files labeled the same as the folders? or to download them in one folder with the same labels that appear in EDGAR (e.g. for MSFT, the 10-K filing final submission for 2021 is  0001564590-21-039151.txt)

https://www.sec.gov/Archives/edgar/data/0000789019/000156459021039151/0001564590-21-039151-index.htm

Many thanks.
